code_samples/workflow/services/workflow.yaml

```
@@ -0,0 +1,5 @@
+services:
+    App\Checkout\Workflow\Strategy\NewWorkflow:
+        tags:
+            - name: ibexa.checkout.workflow.strategy
+              priority: -80
```

For custom code is recommended to use higher values

```suggestion
              priority: 100
```